RxCocoa/iOS/WKWebView+Rx.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  WKWebView+Rx.swift
+//  RxCocoa
+//
+//  Created by Giuseppe Lanza on 14/02/2020.
+//  Copyright © 2020 Krunoslav Zaher. All rights reserved.
+//
+
+#if os(iOS) || os(macOS)
+
+import RxSwift
+import WebKit
+
+@available(iOS 8.0, OSX 10.10, OSXApplicationExtension 10.10, *)
+extension Reactive where Base: WKWebView {
+    
+    /// Reactive wrapper for `navigationDelegate`.
+    /// For more information take a look at `DelegateProxyType` protocol documentation.
+    public var navigationDelegate: DelegateProxy<WKWebView, WKNavigationDelegate> {
+        RxWKNavigationDelegateProxy.proxy(for: base)
+    }
+    
+    /// Reactive wrapper for `navigationDelegate` message.
+    public var didCommit: Observable<WKNavigation> {
+        navigationDelegate
+            .methodInvoked(#selector(WKNavigationDelegate.webView(_:didCommit:)))
+            .map { a in try castOrThrow(WKNavigation.self, a[1]) }
+    }
+    
+    /// Reactive wrapper for `navigationDelegate` message.
+    public var didStartLoad: Observable<WKNavigation> {
+        navigationDelegate
+            .methodInvoked(#selector(WKNavigationDelegate.webView(_:didStartProvisionalNavigation:)))
+            .map { a in try castOrThrow(WKNavigation.self, a[1]) }
+    }
+
+    /// Reactive wrapper for `navigationDelegate` message.
+    public var didFinishLoad: Observable<WKNavigation> {
+        navigationDelegate
+            .methodInvoked(#selector(WKNavigationDelegate.webView(_:didFinish:)))
+            .map { a in try castOrThrow(WKNavigation.self, a[1]) }
+    }
+
```

Comment:
I'm confused why this is here and part of your PR - it seems like you're not rebased correctly

Reply:
Because I’m not sure which branch you would like this based off of. The only one I found is a few commits behind the tag this is based off of, so they’re included in the PR